packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -5214,9 +5256,13 @@ type StylesheetResource = {
   state: StylesheetState,
 };
 
-export type BoundaryResources = {
+export type BoundaryState = {
   styles: Set<StyleQueue>,
   stylesheets: Set<StylesheetResource>,
+  // Hoistable chunks
+  charsetChunks: Array<Chunk | PrecomputedChunk>,
+  viewportChunks: Array<Chunk | PrecomputedChunk>,
```

Comment:
Planning to move charset to config to avoid all this plumbing for something that should always be at the root and doesn't compose. Viewport is harder because it doesn't have to go at the top even though you'd definitely want it to.